helix-term/src/commands.rs

```diff
@@ -4701,6 +4701,7 @@ fn select_register(cx: &mut Context) {
         if let Some(ch) = event.char() {
             cx.editor.autoinfo = None;
             cx.editor.selected_register = Some(ch);
+            cx.editor.set_status(format!("Selected register [{}]", ch));
```

Comment:
I think that we don't need the status message if we also will show the selected register in the statusline